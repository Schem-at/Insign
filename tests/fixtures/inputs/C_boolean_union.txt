@base=rc([0,0,0],[7,7,7])
@extension=rc([8,0,0],[15,7,7])
@combined=base+extension
#combined:type="union"
#combined:components=["base","extension"]
