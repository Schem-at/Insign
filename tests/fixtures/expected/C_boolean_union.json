{
  "base": {
    "bounding_boxes": [
      [
        [
          0,
          0,
          0
        ],
        [
          7,
          7,
          7
        ]
      ]
    ],
    "metadata": {}
  },
  "combined": {
    "bounding_boxes": [
      [
        [
          0,
          0,
          0
        ],
        [
          7,
          7,
          7
        ]
      ],
      [
        [
          8,
          0,
          0
        ],
        [
          15,
          7,
          7
        ]
      ]
    ],
    "metadata": {
      "components": [
        "base",
        "extension"
      ],
      "type": "union"
    }
  },
  "extension": {
    "bounding_boxes": [
      [
        [
          8,
          0,
          0
        ],
        [
          15,
          7,
          7
        ]
      ]
    ],
    "metadata": {}
  }
}